FEATURES.md

```diff
@@ -0,0 +1,17 @@
+# Proposing Features
+
+A feature request or proposal is an issue opened on a repo in opensearch-project that states an end-user problem, and proposes a solution in the form of a feature. It should meet the following criteria.
+
+- Is a feature.
+- Fits in the scope of the repo and project.
+- Is sufficiently distinct from existing features.
+- Doesn’t overlap or conflict with ongoing work.
+- Is sufficiently atomic.
```

Comment:
This might need to have a few more words around it, particularly the "sufficiently" part.  Is there a litmus test or example you could provide?

Reply:
I will remove both sufficiently.